docker-compose.yml

```diff
@@ -1,31 +1,34 @@
 version: "3"
 services:
-  app:
-    image: dm_app_image
-    container_name: dm_app
-    build:
-      context: ././
-      dockerfile: src/front-end/Dockerfile
-    ports:
-      - "3000:3000"
-    environment:
-      POSTGRES_URL: postgresql://digitalmarketplace:digitalmarketplace@db:5432/digitalmarketplace
-      NODE_ENV: ${NODE_ENV}
-    depends_on:
-      - db
+  # app:
+  #   image: dm_app_image
+  #   container_name: dm_app
+  #   build:
+  #     context: ././
+  #     dockerfile: src/front-end/Dockerfile
+  #   ports:
+  #     - "3000:3000"
+  #   environment:
+  #     POSTGRES_URL: postgresql://digitalmarketplace:digitalmarketplace@db:5432/digitalmarketplace
+  #     NODE_ENV: ${NODE_ENV}
+  #   depends_on:
+  #     - db
```

Comment:
I think I'd ultimately like to delete this--I don't want to run the app in a container when I'm developing locally; I want to run it with watch

Reply:
Local dev is the only reason this exists in the first place right?

Reply:
Yes